CHANGELOG.md

```diff
@@ -1,11 +1,20 @@
+## 10.0.0-BETA.9 (2020-XX-XX)
+
+### Enhancements
+* Users can now opt out from allowing queries to be launched from the UI thread by using `RealmConfiguration.Builder.allowQueriesOnUiThread(false)`. Doing so will result in a `RealmException` when calling `RealmQuery.findAll()` or `RealmQuery.findFirst()`. Queries will still be allowed from any thread by default, although we recommend using a non-UI thread to launch queries, or, alternatively, using `RealmQuery.findAllAsync()` or `RealmQuery.findFirstAsync()`
+
+### Breaking Changes
+* From now on it is not allowed by default to run transactions with `Realm.executeTransaction` from the UI thread, resulting in a `RealmException`. Users can override this setting by using `RealmConfiguration.Builder.allowWritesOnUiThread(true)` when building a `RealmConfiguration` to obtain a Realm instance, though we do not recommend doing so. Instead, we recommend using non-UI threads when calling `Realm.executeTransaction`, or, alternatively, using `Realm.executeTransactionAsync`.
+
+
 ## 10.0.0-BETA.8 (2020-09-23)
 
 We no longer support Realm Cloud (legacy), but instead the new MongoDB Realm Cloud. MongoDB Realm is a serverless platform that enables developers to quickly build applications without having to set up server infrastructure. MongoDB Realm is built on top of MongoDB Atlas, automatically integrating the connection to your database.
 
 The old Realm Cloud legacy APIs have undergone significant refactoring. The new APIs are all located in the `io.realm.mongodb` package with `io.realm.mongodb.App` as the entry point.
 
 ### Fixed
-* [RealmApp] Logging in caused an `token contains an invalid number of segments` error. (Issue [#7117](https://github.com/realm/realm-java/issues/7117), since 10.0.0-BETA.7) 
+* [RealmApp] Logging in caused an `token contains an invalid number of segments` error. (Issue [#7117](https://github.com/realm/realm-java/issues/7117), since 10.0.0-BETA.7)
```

Comment:
I've no idea why this appears as a modification. I didn't change anything here 🤷

Reply:
I see. The whitespace at the end got removed automatically.